Feedback/Helper/Mail.php

```diff
@@ -0,0 +1,106 @@
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\Translate\Inline\StateInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\Framework\Escaper;
+
+/**
+ * Feedback Mail Helper
+ *
+ * @package Tasks\Feedback\Helper
+ */
+class Mail extends AbstractHelper
+{
+    /**
+     * @var TransportBuilder
+     */
+    protected $transportBuilder;
+
+    /**
+     * @var StoreManagerInterface
+     */
+    protected $storeManager;
+
+    /**
+     * @var Escaper
+     */
+    protected $escaper;
+
+    /**
+     * @var StateInterface
+     */
+    protected $inlineTranslation;
+
+    /**
+     * Mail constructor.
+     *
+     * @param Context $context
+     * @param TransportBuilder $transportBuilder
+     * @param StoreManagerInterface $storeManager
+     * @param Escaper $escaper
+     * @param StateInterface $state
+     */
+    public function __construct(
+        Context $context,
+        TransportBuilder $transportBuilder,
+        StoreManagerInterface $storeManager,
+        Escaper $escaper,
+        StateInterface $state
+    ) {
+        $this->escaper = $escaper;
+        $this->transportBuilder = $transportBuilder;
+        $this->storeManager = $storeManager;
+        $this->inlineTranslation = $state;
+        parent::__construct($context);
+    }
+
+    /**
+     * Send email with the specified template
+     *
+     * @param string $email
+     * @param string $templateId
+     */
+    public function sendEmail($email, $templateId)
+    {
+        // Example code for sending email
+        // You can customize template ID, sender email, receiver email, and template variables as needed.
+
+        $fromEmail = 'dharundash@gmail.com';  // Sender's Email address
+        $fromName = 'Admin';             // Sender's Name
+        $toEmail = $email; // Receiver's email address
+
+        try {
+            // Template variables to pass
+            $templateVars = [
+                'AcceptStatus' => 'accepted',
+                'RejectStatus' => 'rejected'
+            ];
+
+            $storeId = $this->storeManager->getStore()->getId();
+
+            $from = ['email' => $fromEmail, 'name' => $fromName];
+            $this->inlineTranslation->suspend();
+
+            $storeScope = \Magento\Store\Model\ScopeInterface::SCOPE_STORE;
+            $templateOptions = [
+                'area' => \Magento\Framework\App\Area::AREA_ADMINHTML,
+                'store' => $storeId
+            ];
+            $transport = $this->transportBuilder->setTemplateIdentifier($templateId)
+                ->setTemplateOptions($templateOptions)
+                ->setTemplateVars($templateVars)
+                ->setFrom($from)
+                ->addTo([$toEmail])
+                ->getTransport();
+            $transport->sendMessage();
+            $this->inlineTranslation->resume();
+        } catch (\Exception $e) {
+            $this->_logger->info($e->getMessage());
```

Comment:
```suggestion
            $this->_logger->error($e->getMessage());
```
Better to use the error method here. Because It's an exception.

Reply:
Please follow the same on other applicable places as well.